internal/services/applications/application_resource.go

```diff
@@ -225,6 +274,51 @@ func applicationResource() *schema.Resource {
 				},
 			},
 
+			"info": {
```

Comment:
info seems superfluous - should we just move these all to the top level isntead?

Reply:
Agreed, that makes sense